### PR TITLE
writer title

### PR DESCRIPTION
**Pull request by valerii-fed**
body

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,21 +3,21 @@ test1
 test2
 test3
 test4
-test5
+
```
Comment (valerii-fed) [inline README.md]: it's a new comment (val-fed)

Comment (valerii-fed) [inline README.md]: it's a new comment (val-fed)

File: README.md [hunk 2]
```diff
@@ -3,21 +3,21 @@ test1
...
 test6
 test7
 test8
 test9
 test10
-test11
+edit-test11
 test12
 test13
 test14
```
Comment (valerii-fed) [inline README.md]: iyujthygtfd

File: README.md [hunk 3]
```diff
@@ -3,21 +3,21 @@ test1
 test2
 test3
 test4
-test5
+
 test6
```
Comment (valerii-fed) [inline README.md]: ....

File: README.md [hunk 4]
```diff
@@ -3,21 +3,21 @@ test1
 test2
 test3
```
Comment (valerii-fed) [inline README.md]: a new comment (val-fed)

File: test.txt [hunk 5]
```diff
@@ -1,20 +1,17 @@
...
+
 test13
-test14
 test15
 test16
-test17
+edit-test17
 test18
 test19
 test20
```
Comment (valerii-fed) [inline test.txt]: iuuykuykk

File: test.txt [hunk 6]
```diff
@@ -1,20 +1,17 @@
-test1
-test2
 test3
 test4
-test5
+edit
 test6
-test7
 test8
```
Comment (valerii-fed) [inline test.txt]: edited comment

File: README.md (hunk 3 above)
Comment (val-fed) [reply README.md]: val-fed comment

**Pull request closed without merging**